express/src/routes/manage-account.ts

```diff
@@ -68,7 +68,7 @@ router.post(
     verifyMobileWithSmsCode
 );
 
-router.get("/change-service-name/:serviceId/:selfServiceClientId/:clientId", checkAuthorisation, (req, res) => {
+router.get("/account/change-service-name/:serviceId/:selfServiceClientId/:clientId", checkAuthorisation, (req, res) => {
```

Comment:
```suggestion
router.get("/service/:serviceId/client/:clientId/:selfServiceClientId/change-service-name", checkAuthorisation, (req, res) => {
```
Follow the same pattern we use for services and clients